superset-frontend/src/components/Icons/AntdEnhanced.tsx

```diff
@@ -17,10 +17,94 @@
  * under the License.
  */
 
-import * as AntdIcons from '@ant-design/icons/lib/icons';
+import {
+  AlignCenterOutlined,
+  AlignLeftOutlined,
+  AlignRightOutlined,
+  ApartmentOutlined,
+  BarChartOutlined,
+  BookOutlined,
+  CalendarOutlined,
+  CheckOutlined,
+  CheckSquareOutlined,
+  CloseOutlined,
+  ConsoleSqlOutlined,
+  CopyOutlined,
+  DatabaseOutlined,
+  DeleteFilled,
+  DownOutlined,
+  EditOutlined,
+  ExclamationCircleOutlined,
+  EyeOutlined,
+  EyeInvisibleOutlined,
+  FallOutlined,
+  FileImageOutlined,
+  FileOutlined,
+  FireOutlined,
+  FullscreenExitOutlined,
+  FullscreenOutlined,
+  FundProjectionScreenOutlined,
+  InfoCircleOutlined,
+  InsertRowBelowOutlined,
+  MonitorOutlined,
+  PlusCircleOutlined,
+  PlusOutlined,
+  ReloadOutlined,
+  RightOutlined,
+  SaveOutlined,
+  SearchOutlined,
+  SettingOutlined,
+  StopOutlined,
+  TagsOutlined,
+  UpOutlined,
+  UserOutlined,
+} from '@ant-design/icons';
 import { StyledIcon } from './Icon';
 import IconType from './IconType';
 
+const AntdIcons = {
```

Comment:
I looked for ways to keep things DRY here and not have to repeat icon names twice, but didn't find any way. Too bad ECMAScript doesn't support something like `import {a, b, c} as container from ....`